### PR TITLE
rtmp-services: Add Dolby Millicast

### PR DESCRIPTION
**Pull request by FabienLavocat**
### Description
Add [Dolby Millicast](https://dolby.io) to RTMP Services list.

### Motivation and Context
Add support for "Dolby Millicast" in RTMP Services.

### How Has This Been Tested?
Local Build & CI Build

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:

- [X] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [X] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [X] The code has been tested.
- [X] All commit messages are properly formatted and commits squashed where appropriate.
- [X] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2922,6 +2922,102 @@
...
+                {
+                    "name": "Sydney, Australia (RTMPS)",
+                    "url": "rtmps://rtmp-syd-1.millicast.com:443/v2/pub"
+                },
+                {
+                    "name": "Sydney, Australia (RTMP)",
+                    "url": "rtmp://rtmp-syd-1.millicast.com:1935/v2/pub"
+                }
+            ],
+            "protocol": "RTMP",
```
Comment (tytan652) [inline plugins/rtmp-services/data/services.json:3013]:
```suggestion
```
The service is RTMP and RTMPS, so the protocol mustn't be set to default to RTMP/RTMPS.

**Review by tytan652 — CHANGES_REQUESTED** [review]

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (FabienLavocat) [reply plugins/rtmp-services/data/services.json]: Thank you for the recommendation, I've removed the `protocol`.

**Review by Fenrirthviti — APPROVED** [review]
Approved from an eligibility standpoint

**Review by derrod — CHANGES_REQUESTED** [review]
Please squash your commmits so there's only one.

**Comment by FabienLavocat** [comment]
@derrod ✅

**Review by RytoEX — APPROVED** [review]
Setting common to false is technically unnecessary, but it's fine. LGTM

**Pull request merged**